src/components/actions/tests/actionBarsReducers.spec.ts

```diff
@@ -67,90 +69,92 @@ describe('Actions', () => {
-
-      oldState = actionBarsState;
-      action.payload.id = 'some-action-bar2';
-      actionBarsState = actionBarsReducer(oldState, action);
-
-      expect(actionBarsState.length).toBe(oldState.length - 1);
-      expect(actionBarsState.filter(actionBar => actionBar.id === action.payload.id).length).toBe(0);
-    });
-
-    it('should return the old state when the action is "REMOVE_ACTION_BAR" and the prompt id does not exist', () => {
-      let oldState: IActionBarState[] = [
-        {
-          id: 'some-action-bar2',
-          actions: undefined
-        }, {
-          id: 'some-action-bar',
-          actions: undefined
-        }, {
-          id: 'some-action-bar3',
-          actions: undefined
-        }
-      ];
-      let action: IReduxAction<IActionBarPayload> = {
-        type: ActionBarActions.remove,
-        payload: {
-          id: 'some-action-bar4'
-        }
-      };
-      let actionBarsState: IActionBarState[] = actionBarsReducer(oldState, action);
-
-      expect(actionBarsState.length).toBe(oldState.length);
-      expect(actionBarsState.filter(actionBar => actionBar.id === action.payload.id).length).toBe(0);
-    });
-
-    it('should add the model (with its actions) to the action bar when the action is "ADD_ACTIONS"', () => {
-      let oldState: IActionBarState[] = [
-        {
-          id: 'some-action-bar2',
-          actions: undefined
-        }, {
-          id: 'some-action-bar',
-          actions: undefined
-        }, {
-          id: 'some-action-bar3',
-          actions: undefined
-        }
-      ];
-
-      let action: IReduxAction<IChangeActionBarActionsPayload> = {
-        type: ActionBarActions.addActions,
-        payload: {
-          id: 'some-action-bar3',
-          actions: [{ enabled: true }]
-        }
-      };
-
-      let actionBarsState: IActionBarState[] = actionBarsReducer(oldState, action);
-
-      expect(actionBarsState.length).toBe(oldState.length);
-      expect(actionBarsState.filter(actionBar => actionBar.id === action.payload.id)[0].actions).toBeDefined();
-      expect(actionBarsState.filter(actionBar => actionBar.id !== action.payload.id)[0].actions).toBeUndefined();
+    describe('with action bars in the state', () => {
+      let oldState: IActionBarState[];
+
+      beforeEach(() => {
+        oldState = [
+          {
+            id: 'some-action-bar2',
+            actions: undefined
+          }, {
+            id: 'some-action-bar',
+            actions: undefined
+          }, {
+            id: 'some-action-bar3',
+            actions: undefined
+          }
+        ];
+      });
+
+      it('should return the old state without the PromptState with the prompt id when the action is "REMOVE_ACTION_BAR', () => {
+        const action: IReduxAction<IActionBarPayload> = {
+          type: ActionBarActions.remove,
+          payload: {
+            id: 'some-action-bar'
```

Comment:
`id: oldState[1].id` will avoid issue on refactoring (same thing for the next ids, you should use the value from the array)